.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -1,6 +1,9 @@
 name: inductor-A100-perf-nightly
 
 on:
+  pull_request:
```

Comment:
Btw, I guess you add this for testing, usually, the way people use this is to manually dispatch the workflow using https://github.com/pytorch/pytorch/actions/workflows/inductor-perf-test-nightly.yml pointing to their branch

Reply:
https://github.com/pytorch/pytorch/actions/runs/11672243539 test jobs are all skipped, let me tweak this PR a bit and rerun

Reply:
precisely, just figured that it might be easier if I needed to iterate over it.

Thanks for helping me test this :)